WordPressAuthenticator/GoogleSignIn/Data+SHA256.swift

```diff
@@ -0,0 +1,20 @@
+import CommonCrypto
+
+extension Data {
+
+    func sha256Hashed() -> Data {
+        Data(sha256Hash())
+    }
+
+    func sha256Hashed() -> String {
+        sha256Hash().map { String(format: "%02x", $0) }.joined(separator: "")
+    }
+
+    private func sha256Hash() -> [UInt8] {
+        var hash = [UInt8](repeating: 0, count: Int(CC_SHA256_DIGEST_LENGTH))
+        withUnsafeBytes {
```

Comment:
This is probably fine, but anytime I see `unsafe` in the code I like to think about if there's a better way.

[CryptoKit has a `SHA256` implementation](https://developer.apple.com/documentation/cryptokit/sha256) (that probably just does this under the hood 😂), but that wouldn't require us to use any `unsafe` code. I believe it's within our minimum iOS version so we should be ok to use it. WDYT?

Reply:
🤦‍♂️ an to think CryptoKit had been available all along...

Great idea, thanks. The implementation also ends up more straightforward.

See 3051ce42